tcf_website/management/commands/upload_csv_review_drive.py

```diff
@@ -0,0 +1,174 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from tcf_website.models import *
+from datetime import datetime
+import csv
+import os
+
+DATA_DIR = "tcf_website/management/commands/review_drive_responses/"
+
+class Command(BaseCommand):
+
+    # Run this coomand using `sudo docker exec -it tcf_django python3 manage.py upload_csv_review_drive [filename] [season] [year]`
```

Comment:
Why is this here - I don't think we should comment stuff like this.

I've done quite a lot in the past but I think if a dev hasn't learn how Django commands work (and how to user docker) there's a fundamental lack of understanding somewhere else.

Reply:
I think it's fine to leave as is, many of the other commands have this in comments, I see no reason to not follow consistent formatting of commands.